### PR TITLE
HOTFIX: Add the missing quotation for livy cancel

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
Seems I miss the quote when rebase. This cause the session recreation fails, I correct it in this PR. 

### What type of PR is it?
[Hot Fix |]

### Todos
* [ ] - Task

### What is the Jira issue?
No jira created, just minor change for hotfix

### How should this be tested?
Tested manually 

### Screenshots (if appropriate)
image

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java [hunk 1]
```diff
@@ -227,6 +220,13 @@ private SessionInfo createSession(String user, String kind)
         }
         Thread.sleep(pullStatusInterval);
         sessionInfo = getSessionInfo(sessionInfo.id);
+        LOGGER.info("Session {} is in state {}, appId {}", sessionInfo.id, sessionInfo.state,
+            sessionInfo.appId);
+        if (sessionInfo.isFinished()) {
+          String msg = "Session " + sessionInfo.id + " is finished, appId: " + sessionInfo.appId
+              + ", log: " + sessionInfo.log;
+          throw new LivyException(msg);
+        }
```
Comment (zjffdu) [inline livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java]: Just move the logging for more clean logging.

**Comment by zjffdu** [comment]
@minahlee Could you merge it into 0.7 if you plan for another RC ? Very appreciated.

File: livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java (hunk 1 above)
Comment (jongyoul) [inline livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java]: You want to reduce the logs while looping, don't you?

Comment (zjffdu) [inline livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java]: Not exactly, just want to the log the last status so I move them after this line

```
sessionInfo = getSessionInfo(sessionInfo.id);
```

File: livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java [hunk 2]
```diff
@@ -438,7 +438,7 @@ private String callRestAPI(String targetURL, String method, String jsonData)
         || response.getStatusCode().value() == 201) {
       return response.getBody();
     } else if (response.getStatusCode().value() == 404) {
-      if (response.getBody().matches("Session '\\d+' not found.")) {
+      if (response.getBody().matches("\"Session '\\d+' not found.\"")) {
```
Comment (zjffdu) [inline livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java]: add the missing quote here.

**Pull request closed without merging**